PlayCover/Utils/SaveGenshinUserData.swift

```diff
@@ -56,13 +60,12 @@ func storeUserData( folderName: String, accountRegion: String ) {
 
             try fileManager.copyItem(at: lastAccountModelEncryptUrl,
                                      to: URL(fileURLWithPath: storePath + lastAccountModelEncrypt))
-            fileManager.createFile(atPath: storePath + "region.txt", contents: nil, attributes: nil)
-            try accountRegion.write(to: URL(fileURLWithPath: storePath + "region.txt"),
-                                    atomically: false, encoding: .utf8)
-
-        } catch {
-            Log.shared.error("Error moving file: \(error)")
-        }
+            if BUNDLE_ID == "com.miHoYo.GenshinImpact" {
+                fileManager.createFile(atPath: storePath + "region.txt", contents: nil, attributes: nil)
+                try accountRegion.write(to: URL(fileURLWithPath: storePath + "region.txt"),
+                                        atomically: false, encoding: .utf8)
+            }
+        } catch { Log.shared.error("Error moving file: \(error)") }
```

Comment:
```suggestion
        } catch {
            Log.shared.error("Error moving file: \(error)") }
        }
```
I think this line should be modified to adjust to the styling in the code.

Reply:
It is common pattern found in the app, it is mostly used to reduce body lines.